Components/Hlms/Pbs/include/Cubemaps/OgreCubemapProbe.h

```diff
@@ -229,7 +241,7 @@ namespace Ogre
         const Matrix3& getInvOrientation(void) const        { return mInvOrientation; }
         const Aabb& getProbeShape(void) const               { return mProbeShape; }
 
-        CompositorWorkspace *getWorkspace(void) const       { return mWorkspace; }
+        CompositorWorkspace **getWorkspaces(void)           { return mWorkspaces; }
```

Comment:
What return type do you prefer @darksylinc? A pointer of pointers or passing the workspace index as a parameter and return a simple pointer?